osf/utils/workflows.py

```diff
@@ -355,6 +378,57 @@ def db_name(self):
     },
 ]
 
+
+COLLECTION_SUBMISSION_TRANSITIONS = [
+    {
+        'trigger': 'submit',
+        'source': [CollectionSubmissionStates.IN_PROGRESS],
+        'dest': CollectionSubmissionStates.ACCEPTED,
+        'before': [],
+        'after': ['_on_submit'],
+    },
+    {
+        'trigger': 'submit',
+        'source': [CollectionSubmissionStates.IN_PROGRESS],
+        'dest': CollectionSubmissionStates.PENDING,
+        'before': [],
+        'after': ['_on_submit'],
+        'conditions': ['is_moderated'],
+    },
```

Comment:
Pretty sure the order on these needs to be reversed, otherwise the state machine will just see the first supported `submit` transition and take it. This is where tests would be useful to know for sure.

Reply:
Ah, I will use `'unless': ['is_moderated']` for the unmoderated flow